packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -38,19 +43,34 @@ const ConfirmYourEmail = observer(() => {
     return (
         <PhoneVerificationCard is_small_card>
             <Text bold>
-                <Localize i18n_default_text="Confirm it's you" />
+                {is_email_verified ? (
```

Comment:
what happens when email is not verified but we have to show verification email page. Example, there could be some issue and we didn't receive the email. How would we handle that scenario?

Reply:
If email is not verified we should always show Email Verification Page for users. This will handle the scenario as user will not get their phone verified if they didn't pass the email verification. But still i need to clarify it with felipe about verification for email.

Reply:
I've checked with felipe so there will be a call for `verify_email` instead of using the phone_number_verificaiton from get settings, we should use the data returned from the `verify_email` API call. Will edit the code here 🙏

Reply:
Wonderful! So, do we add any loader or anything here? while we still wait for the verify_email response? It's kind of related with resend-code-timer since we don't know when to start the counter 🤔

Reply:
For now I'll just create a store value for it and mock the way we can display the phone verification and email verification otp

Reply:
Cool, let's discuss this later 🙏